### PR TITLE
fix: (integrations) add missing icons on fr/ja and missing titles on fr

### PR DESCRIPTION
**Pull request by Flosciante**

### CODE REVIEW

File: content/fr/integrations/deployments/21yunbox.md [hunk 1]
```diff
@@ -4,7 +4,15 @@ title: 21YunBox
 description: Comment déployer Nuxt sur 21YunBox ?
 target: Static
 category: deployment
-logo: "/img/modules/categories/deployment.svg"
+logo:
+  light: "/img/partners/dark/Yunbox.svg"
+  dark: "/img/partners/light/Yunbox.svg"
+---
+
+# Deployer Nuxt sur 21YunBox
```
Comment (smarroufin) [inline content/fr/integrations/deployments/21yunbox.md:12]: D**é**ployer

File: content/fr/integrations/deployments/bip.md [hunk 2]
```diff
@@ -4,7 +4,14 @@ title: Bip
 description: Comment déployer une application Nuxt avec Bip ?
 target: Static
 category: deployment
-logo: "/img/modules/categories/deployment.svg"
+logo:
+  light: "/img/partners/dark/bip.png"
+  dark: "/img/partners/light/bip.png"
+---
+# Deployer Nuxt avec Bip
```
Comment (smarroufin) [inline content/fr/integrations/deployments/bip.md:11]: D**é**ployer

File: content/fr/integrations/deployments/cleavr.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
-description: Comment déployer une application Nuxt avec Cleavr?
+description: Comment déployer une application Nuxt avec Cleavr ?
 target: Static & Server
 category: deployment
 logo:
   light: "/img/partners/dark/cleavr.svg"
   dark: "/img/partners/light/cleavr.svg"
 ---
-
 # Deploy Nuxt with Cleavr
```
Comment (smarroufin) [inline content/fr/integrations/deployments/cleavr.md:11]: Déployer Nuxt sur Cleavr

File: content/fr/integrations/deployments/nginx.md [hunk 4]
```diff
@@ -4,7 +4,14 @@ title: NGINX
...
 target: Static & Server
 category: deployment
-logo: "/img/modules/categories/deployment.svg"
+logo:
+  light: "/img/partners/dark/Nginx.svg"
+  dark: "/img/partners/light/Nginx.svg"
+---
+# Using NGINX as a reverse proxy
+
+How to use nginx as a reverse proxy
```
Comment (smarroufin) [inline content/fr/integrations/deployments/nginx.md:13]: "?"

File: content/fr/integrations/deployments/surge.md [hunk 5]
```diff
@@ -4,7 +4,14 @@ title: Surge
...
 target: Static
 category: deployment
-logo: "/img/modules/categories/deployment.svg"
+logo:
+  light: "/img/partners/dark/Surge.svg"
+  dark: "/img/partners/light/Surge.svg"
+---
+# Déployer Nuxt vec Surge
+
+Comment deployer Nuxt.js sur Surge ?
```
Comment (smarroufin) [inline content/fr/integrations/deployments/surge.md:13]: d**é**ployer

**Review by smarroufin — CHANGES_REQUESTED** [review]

File: content/fr/integrations/deployments/nginx.md (hunk 4 above)
Comment (smarroufin) [reply content/fr/integrations/deployments/nginx.md:13]: Actually, this should be in french

**Review by smarroufin — APPROVED** [review]

**Review by pi0 — COMMENTED** [review]
Since adding more `logo: { light, dark }` can you please check https://github.com/nuxtlabs/nuxtjs.org-board/issues/207 ? Error happens with `nuxt generate`

**Review by Atinux — APPROVED** [review]

**Pull request merged**